CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
   Neovim.
 - Fixed a bug where hovering over an expression in the middle of a pipe would
   give the wrong node.
+- Go to definition now works for types in external Gleam modules.
```

Comment:
This says types, but the test is for a value. Could we have a test for types too in that case.

---


External modules are Erlang or JavaScript modules
```suggestion
- Go to definition now works for types in dependency Gleam modules.
```

Reply:
This was a mistype on my part. It looks like this will only work for values (as far as I can tell goto type does not work for the actual type even in the same file rn)

Reply:
But lemme add a test for record constructors/constants/etc at least to show things besides functions

Reply:
Fab! Thank you